gating/check/run_elk_tests.sh

```diff
@@ -32,5 +32,5 @@ source "$(readlink -f $(dirname ${0}))/../mnaio_vars.sh"
 
 ${MNAIO_SSH} <<EOS
   cd /opt/rpc-openstack
-  openstack-ansible playbooks/elk-deployment.yml
+  openstack-ansible playbooks/site-logging.yml
```

Comment:
renaming 'elk-deployment.yml', so shouldn't we use 'deployment-elk.yml' here?

Reply:
`site-logging.yml` takes care of some housekeeping and then calls `deployment-elk.yml`